Prac06/converting.py

```diff
@@ -0,0 +1,22 @@
+from kivy.app import App
+from kivy.lang import Builder
+from kivy.core.window import Window
+
+class miles_to_kilometres(App):
+    def build(self):
+        Window.size = (400, 300)
+        self.title = "Converting Calculator"
+        self.root = Builder.load_file('converting.kv')
+        return self.root
+
+    def handle_calculate(self, value):
+        result = value * 1.60934
```

Comment:
One thing to think about is putting values like this into a constant, that way your code becomes more readable, and makes it easier to make edits later on (obviously this value is unlikely to be swapped out with anything, but this applies to all constant values in all programs)